src/shared/components/query/GeneticProfileSelector.tsx

```diff
@@ -150,7 +150,7 @@ export default class GeneticProfileSelector extends QueryStoreComponent<{}, {}>
 		{
 			output.push(
 				<div key={output.length} className={styles.zScore}>
-					Enter a z-score threshold ±:
+					Enter a z-score threshold &plusmn;:

```

Comment:
@adamabeshouse This doesn't seem to fix it. When I use the build from jitpack it still shows up as  Â±:. we should maybe do another CSS hack, @alisman ?